content/src/service/ServiceImpl.ts

```diff
@@ -1,10 +1,11 @@
+import { ContentItem } from '@dcl/catalyst-storage'
 import { AuthChain, Authenticator } from '@dcl/crypto'
 import { Entity, EntityType, IPFSv2 } from '@dcl/schemas'
 import { ILoggerComponent } from '@well-known-components/interfaces'
 import { EnvironmentConfig } from '../Environment'
 import { runReportingQueryDurationMetric } from '../instrument'
 import { calculateDeprecatedHashes, calculateIPFSHashes } from '../logic/hashing'
-import { bufferToStream, ContentItem } from '../ports/contentStorage/contentStorage'
+import { bufferToStream } from '@dcl/catalyst-storage/dist/content-item'
```

Comment:
Is it necessary to import from `.../dist/...`?

Reply:
ts is nasty sometimes.. having a src + dist folders is the best way I've found to deal with conflicts. since bufferToStream and SimpleContentItem are internal utils for testing, those are not exported by index.ts

Reply:
But if we're using them here, they're not only for internal utils 🤔